lib/chrome-binary-phases.js

```diff
@@ -0,0 +1,18 @@
+var register = require('../core/phase-register.js');
```

Comment:
May I suggest moving this fellow into lib/chromium-phases/chromium-phases.js? It's a nice spot there :smiley: